temporalcli/commandsmd/commands.md

```diff
@@ -655,7 +663,107 @@ The temporal task-queue list-partition command displays the partitions of a Task
 
 * `--task-queue`, `-t` (string) - Task queue name. Required.
 
-### temporal task-queue update-build-ids: Operations to update the sets of worker Build ID versions on the Task Queue.
+
+### temporal task-queue update-build-id-rules: Updates the worker Build ID assignment and redirect rules on the Task Queue.
+
+Provides various commands for adding, removing, or replacing worker Build ID assignment and redirect rules associated with a Task Queue. See the help of each sub-command for more.
+
+### temporal task-queue update-build-id-rules add-redirect-rule: Adds the rule to the list of redirect rules for this Task Queue.
```

Comment:
That's a mouthful. Maybe we should change `update-build-id-rules` to just `versioning` or `build-ids` or something and put the `get-rules` under it instead of adjacent.

Reply:
I like that, let's use `versioning` as a top level command, @ShahabT any objections?

Reply:
I too prefer the briefer form.